### ISSUE TITLE
[Bug]: TPCH Q9 Q12 Q19 ERROR internal error: Alloc size 2147483648 too large

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#5946]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.6.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

Now start to execute the query queries/q10.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q10.sql  has been executed successfully,and cost: 12021
Now start to execute the query queries/q11.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q11.sql  has been executed successfully,and cost: 5634
Now start to execute the query queries/q12.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 20101 (HY000) at line 1: internal error: Alloc size 2147483648 too large
The query queries/q12.sql  has been executed successfully,and cost: 8748
Now start to execute the query queries/q13.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q13.sql  has been executed successfully,and cost: 112915
Now start to execute the query queries/q14.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q14.sql  has been executed successfully,and cost: 1536
Now start to execute the query queries/q15.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q15.sql  has been executed successfully,and cost: 1216
Now start to execute the query queries/q16.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q16.sql  has been executed successfully,and cost: 10945
Now start to execute the query queries/q17.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q17.sql  has been executed successfully,and cost: 28043
Now start to execute the query queries/q18.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 20101 (HY000) at line 1: internal error: Alloc size 2147483648 too large
The query queries/q18.sql  has been executed successfully,and cost: 9246
Now start to execute the query queries/q19.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q19.sql  has been executed successfully,and cost: 6976
Now start to execute the query queries/q1.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q1.sql  has been executed successfully,and cost: 6461
Now start to execute the query queries/q20.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q20.sql  has been executed successfully,and cost: 22261
Now start to execute the query queries/q21.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 20101 (HY000) at line 1: internal error: Alloc size 2147483648 too large
The query queries/q21.sql  has been executed successfully,and cost: 89155
Now start to execute the query queries/q22.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q22.sql  has been executed successfully,and cost: 10562
Now start to execute the query queries/q2.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q2.sql  has been executed successfully,and cost: 657
Now start to execute the query queries/q3.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q3.sql  has been executed successfully,and cost: 1450
Now start to execute the query queries/q4.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 20101 (HY000) at line 1: internal error: Alloc size 2147483648 too large
The query queries/q4.sql  has been executed successfully,and cost: 39227
Now start to execute the query queries/q5.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q5.sql  has been executed successfully,and cost: 10548
Now start to execute the query queries/q6.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q6.sql  has been executed successfully,and cost: 1892
Now start to execute the query queries/q7.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 20101 (HY000) at line 1: internal error: Alloc size 2147483648 too large
The query queries/q7.sql  has been executed successfully,and cost: 27319
Now start to execute the query queries/q8.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
The query queries/q8.sql  has been executed successfully,and cost: 2804
Now start to execute the query queries/q9.sql,please wait.....
mysql: [Warning] Using a password on the command line interface can be insecure.
ERROR 20101 (HY000) at line 1: internal error: Alloc size 2147483648 too large
The query queries/q9.sql  has been executed successfully,and cost: 8113

**Comment by m-schen** [comment]
the same to #6178

**Comment by aressu1985** [comment]
fixed:e23e00f1d01d43e33c509a6d239ccc38b5f93409
